homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/gravatar/components/NewAvatarButton.kt

```diff
@@ -35,8 +35,12 @@ fun NewAvatarButton(
     Surface(
         onClick = onClick,
         contentColor = MaterialTheme.colorScheme.primary,
+        color = if (isSystemInDarkTheme()) {
```

Comment:
🤔 I would say that the best part of defining themes is to avoid this kind of `ifs`. I would expect to use the same color at least, for example, `surface`. I know this is not an issue of the PR, just some thoughts.

Reply:
hmm those buttons have completely different styles in dark/light mode so I don't think we can get away from IFs here 🤔 They do have different background colors, and borders, etc.